boards/adi/apard32690/doc/index.rst

```diff
@@ -0,0 +1,220 @@
+.. _ad_apard32690_sl:
+
+AD-APARD32690-SL
+################
+
+Overview
+********
+The AD-APARD32690-SL is a platform for prototyping intelligent, secure, and connected field devices.
+It has an Arduino Mega-compatible form factor and two Pmod-compatible connectors.
+The system includes the MAX32690 ARM Cortex-M4 with FPU-Based Microcontroller and Bluetooth LE 5.2.
+The MCU is coupled with external RAM (2 x 512 Mb) and Flash (64 Mb) memories to meet the requirements
+of the most demanding applications. The MAXQ1065 security coprocessor enables state of the art
+security features such as for root-of-trust, mutual authentication, data confidentiality and
+integrity, secure boot, and secure communications.
+A 10 Mbps single-pair Ethernet link using the ADIN1110 10BASE-T1L MAC/PHY, enables remote
+data acquisition and system configuration. The 10BASE-T1L interface also supports Single-pair
+Power over Ethernet (SPoE) and be used for powering the system via an Arduino shield implementing
+the required power circuitry.
+
+The Zephyr port is running on the MAX32690 MCU.
+
+.. image:: img/apard32690_img1.webp
+   :align: center
+   :alt: AD-APARD32690-SL Front
+
+.. image:: img/apard32690_img1.webp
+   :align: center
+   :alt: AD-APARD32690-SL Back
+
```

Comment:
there's a typo (img1 instead of img2) but I would actually ask that you drop the back image entirely, There's no real value justifying adding 50K to the main git repo. In fact, the fact that no one noticed during review kind of confirms my point, probably :) 

![image](https://github.com/zephyrproject-rtos/zephyr/assets/128251/42dd0c67-d276-416a-a9e2-96576bcc6d75)

Reply:
On a related note, there seems to be better quality images (better lighting at least) available from offical web page https://www.analog.com/en/resources/evaluation-hardware-and-software/evaluation-boards-kits/ad-apard32690-sl.html (almost feels like the ones you used might be from an early prototype? :) )

see e.g. 

![EVAL-MAX32690-ARDZ_ANGLE-web](https://github.com/zephyrproject-rtos/zephyr/assets/128251/4676e790-c179-4bee-82d5-d437011a1c82)

or

![EVAL-MAX32690-ARDZ_TOP-web](https://github.com/zephyrproject-rtos/zephyr/assets/128251/11d3a476-5ee0-48d3-bd60-210795b10f75)

Reply:
Hi @kartben, and thank you for pointing this out. The images which I've added as part of this PR are actually of the current revision of the board, so even though they are of lower quality, I think it would be better to use them instead of the ones from official web page (which is actually the earlier prototype you can't buy anymore). I'll drop the back image.